source/intel.js

```diff
@@ -1977,12 +1977,59 @@ NeptunesPride.npui.StarInspector = function () {
 
   return starInspector;
 };
+//Enable arrows keys to interact with editing a ship's instructions on stay.
+const fleet_order_keyboard_integeration = () => {
+  const event_name = "show_screen";
+
+  const selector_right_button =
+    "#contentArea > div > div.widget.fullscreen > div:nth-child(3) > div > div.widget.rel.col_black > div:nth-child(3)";
+  const selector_left_button =
+    "#contentArea > div > div.widget.fullscreen > div:nth-child(3) > div > div.widget.rel.col_black > div:nth-child(2)";
+  const selector_menu =
+    "#contentArea > div > div.widget.fullscreen > div:nth-child(3) > div > div.widget.rel.col_accent > div.widget.drop_down > select";
+
+  const apply_arrow_click = (selector, key) => {
+    console.log("Event added");
+    const NP_click = (event) => {
+      if (event.keyCode === key) {
+        $(selector).mousedown().mouseup(); //NP does not respond to click
+        console.log(`${key} has been clicked!`);
+      }
+    };
+    $(document).keyup(NP_click);
```

Comment:
I think if you really want to add global hotkeys it should be done with `Mousetrap` instead of hooking the top level document.

Reply:
i actually plan on changing this section later; I didn't write in any comments but I want to add a bit of code to bind only when the page is up then remove the events. 

I spend most of my time & am still stuck with changing the drop-down menu. I have it changing but it does not graphically change on the page.. Very irritating.

Reply:
Yes I can change it to mousetrap. Although, why would I house mousetrap besides that it is what we have been using for hotkeys? 

What actually does it do thats different?

Reply:
The point is probably moot if you liked the approach I took to getting the keys working, as there the events are only hooked locally.

TBH I don't know why Jay used `Mousetrap` or what advantages it offers, but since it is the way of using global hotkeys it seems to me that we should be consistent throughout. Generally developers will monkey-see-monkey-do what we put in the codebase and they should see that a hotkey is normally done like this:
```
  hotkey("*", starReport);
  starReport.help =
    "Generate a report on all stars in your scanning range, and copy it to the clipboard." +
    "<p>This same report can also be viewed via the menu; enter the agent and choose it from the dropdown.";
```
From this they should immediately realize two things: hotkeys are usually bound to existing functions, and hotkeys should have help text. If they monkey-see-monkey-do that, all our hotkeys will be documented and all hotkeys will be bound to named functions in the code. 

It just strikes me as convenient and clean.

Reply:
Using mousetrap seemed to not work the way I wanted. Also attaching to any element other an `document` also seems to not work. 

When the menu object is in focus the left & right keys switching between stars breaks. 

The current solution is insufficient for pulling I am trying to find some new solutions.